File: Desktop/src/hexico/testgame/DesktopStarter.scala
package hexico.testgame

import com.badlogic.gdx.backends.lwjgl.{LwjglApplication, LwjglApplicationConfiguration}

object DesktopStarter extends App {
  val cfg = new LwjglApplicationConfiguration
  cfg.title = "TestGame"
  cfg.useGL20 = true
  cfg.width = 800
  cfg.height = 480
  new LwjglApplication(new TestGame, cfg)
}

File: Desktop/src/hexico/testgame/DesktopStarter.java
package hexico.testgame;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

/**
 * Created with IntelliJ IDEA.
 * User: Alan
 * Date: 24/04/13
 * Time: 19:57
 * To change this template use File | Settings | File Templates.
 */
public class DesktopStarter {
    public static void main(String[] args) {
        LwjglApplicationConfiguration cfg = new LwjglApplicationConfiguration();
        cfg.title = "TestGame";
        cfg.useGL20 = true;
        cfg.width = 800;
        cfg.height = 480;
        new LwjglApplication(new TestGame(), cfg);
    }
}
